.werft/installer-tests.ts

```diff
@@ -369,10 +376,13 @@ export async function installerTests(config: TestConfig) {
 }
 
 function runIntegrationTests() {
-    werft.phase("run-integration-tests", "Run all existing integration tests");
+    const testComponent: string = randomize(["WORKSPACES", "IDE", "WEBAPP"])
```

Comment:
why do we `randomize` here? 🤔

Reply:
The idea is that since these tests are run every night we want one of the three test suites to run each night. Because running all three tests together tend to hit the 2 hour timeout of werft jobs. So in average over a week, it is likely that all tests will run. We can add an annotation to explicitly specify the test suite, eg: `-a test=IDE`, in which case it will take that as `testComponent` instead of calling the `randomize`